README.md

```diff
@@ -1,4 +1,5 @@
-# THE Z SHELL (ZSH)
+
+# <img src="https://www.zsh.org/color_vertical_icon.png" alt="ZSH" width="50"/> THE Z SHELL (ZSH) 
```

Comment:
Our actual project host, SourceForge, [strips raw HTML when rendering Markdown](https://sourceforge.net/p/forge/documentation/Files-Readme/), so this won't work.

I remind you, again, that this project is not hosted on GitHub.

Reply:
I have removed raw HTML and replaced it with a markdown image tag without styling.

FYI Sourceforge does not strip HTML. I may now have to revert the changes 😮‍💨 

https://sourceforge.net/p/zsh-fork/code/ci/a205af4809fbe2335438e6b83c1b55bdabba5504/tree/

Reply:
> FYI Sourceforge does not strip HTML.

I guess their documentation is wrong, or I misunderstood it.

Reply:
That's okay. I will revert back to raw HTML only if we choose to go ahead with the PR. I hope that's alright.